peps/pep-0755.rst

```diff
@@ -0,0 +1,907 @@
+PEP: 755
+Title: TypeExpr: Type Hint for a Type Expression
+Author: David Foster <david at dafoster.net>
+Sponsor: Jelle Zijlstra <jelle.zijlstra at gmail.com>
+Discussions-To: https://discuss.python.org/t/typeform-spelling-for-a-type-annotation-object-at-runtime/51435
```

Comment:
_After_ the PEP has been merged into the repo, this will be the new discussion link.

Reply:
Should I remove the Discussions-To header until the initial merge is complete? Or is there something else I should do with this header?

Reply:
Rereading PEP 1 carefully, I see:

> As soon as a PEP number has been assigned and the draft PEP is committed to the [PEP repository](https://github.com/python/peps), a discussion thread for the PEP should be created to provide a central place to discuss and review its contents, and the PEP should be updated so that the Discussions-To header links to it.

This seems to imply that the original draft PEP must not include a Discussions-To header and it must not include a Post-History. Both of those headers should be created immediately after a _new_ thread for discussing the PEP is created.

So it sounds like I should:
* Delete the Discussions-To and Post-History headers for now, since there's no thread to link to yet.
* Shepherd this PR to merge.
* Create a new discussion thread. Link the old thread to the new thread. Link the PEP to the new thread via a small change adding a Discussions-To and Post-History header.

**Does that sound right** @hugovk / @JelleZijlstra ?

There's already a pretty substantial [discussion thread for drafts 1 and 2](https://discuss.python.org/t/typeform-spelling-for-a-type-annotation-object-at-runtime/51435) of this PEP (before the current draft 3 in this PR). **Should a link to this old thread be (eventually) included in Discussions-To as well?**

Reply:
Yes, that's right. The existing thread should be listed in Post-History.